motif-docs/pages/troubleshooting/configure-codebuild.mdx

```diff
@@ -0,0 +1,23 @@
+---
+heading: Advanced Customization
+title: Customize CodeBuild (CI) Environment CPU & Memory
+---
+import Image from 'next/image'
+import buildLogsButtonImg from '../../public/img/view-build-aws-logs-button.png'
+import codeBuildEditEnv from '../../public/img/codebuild-edit-environment.png'
+import codeBuildEditEnvAdditionalConfig from '../../public/img/codebuild-edit-environment-additional-config.png'
+
+Flightcontrol provisions a CodeBuild project for every service that requires build, this allows different services to build concurrently.
+The standard provisioned project environment (7 GB memory, 4 vCPUs) should be suitable for almost all use cases. If however there is a need to change this configuration:
+
+1. Allocate the CodeBuild project, the easiest way is to click the **View Logs in AWS** button in any Build Logs dialogue.
+<Image className="rounded-primary mx-auto w-full max-w-[250px] border" src={buildLogsButtonImg} />
+2. This will open the CodeBuild project, You will need to go one level up in the project breadcrumps.
+3. Find the project Edit button, then click **Edit Environment**
+<Image className="rounded-primary mx-auto w-full max-w-[250px] border" src={codeBuildEditEnv} />
```

Comment:
make this smaller too, to match the actual size

Reply:
Done